src/components/subscriptions/data/utils.js

```diff
@@ -0,0 +1,9 @@
+import { SEEN_SUBSCRIPTION_EXPIRATION_MODAL_COOKIE_PREFIX } from './constants';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getSubscriptionExpiringCookieName = ({
+  expirationThreshold, enterpriseId,
+}) => {
+  const cookieName = `${SEEN_SUBSCRIPTION_EXPIRATION_MODAL_COOKIE_PREFIX}${expirationThreshold}-${enterpriseId}`;
```

Comment:
Beyond abstracting this out into a utility function, I'm also including the `enterpriseId` so any new cookies set will be unique per enterprise. For example, if I see expiration modal for 60 days out for Enterprise A, I will also see the same expiration modal for Enterprise B.